cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -257,7 +257,9 @@ elseif(ARM OR AARCH64)
   if(NOT AARCH64)
     ocv_update(CPU_KNOWN_OPTIMIZATIONS "VFPV3;NEON;FP16")
     ocv_update(CPU_VFPV3_FLAGS_ON "-mfpu=vfpv3")
-    ocv_update(CPU_NEON_FLAGS_ON "-mfpu=neon")
+    if(NOT MSVC)
+      ocv_update(CPU_NEON_FLAGS_ON "-mfpu=neon")
+    endif()
```

Comment:
> Neon not supported on MSVC ARM

This is [not correct](https://msdn.microsoft.com/en-us/library/hh875058.aspx#Anchor_0).
MSVC doesn't support GNU-like compiler options. So other "-mfpu" flags should be fixed too to keep code consistent.

Do you have CMake toolchain file for MSVC ARM build?

Reply:
I am using the WinRT ARM toolchain file as I do not know of a non-WinRT Windows ARM toolchain file though it would likely be simply modifying the flag in the toolchain to support the full Win API.

You are right.  In fact I got it compiling with all the neon codes too as can be noted from my comment below where a fix to already replaced by universal framework code had to be fixed specifically for neon.  Only this command line was the problem.  I will attempt to generalize it for this file.